lib/plugins/aws/package/compile/events/stream/index.js

```diff
@@ -177,6 +179,7 @@ class AwsCompileStreamEvents {
                     ]
                   },
                   "StartingPosition": "${StartingPosition}",
+                  "MaximumRetryAttempts": ${MaximumRetryAttempts},
```

Comment:
I guess it'll be cleaner to not set this property at all if not explicitly set.

Reply:
Sure, Thanks for the feedback.
But I defaulted it to 10000 (max that AWS) has, don't you think that will suffice ?

Reply:
Is  10000 the default by AWS, or is the default no retries at all? (we need to ensure default behavior, when this is not set)

Even if default is `10000`, then it's cleaner to not pollute template with settings which have no effect.

Reply:
@medikoo I believe the default from AWS is 10000. So this matches default behavior..
Sounds good, I'll make the changes.. Thanks !

Reply:
@medikoo Updated PR as per recommendation. Please review.